modules/airtable_adapter/http_client.py

```
@@ -0,0 +1,55 @@
+    status: int
+    body: str
+
+
+class HttpError(RuntimeError):
+    def __init__(self, *, status: int | None, message: str):
+        super().__init__(message)
+        self.status = status
+
+
+def http_get_json(*, url: str, headers: dict[str, str], timeout_s: float = 30.0) -> dict:
+    req = urllib.request.Request(url, headers=headers, method="GET")
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)